Apps/Common/src/Services/IEmailQueueService.cs

```diff
@@ -79,7 +79,8 @@ public interface IEmailQueueService
         /// <param name="hdid">The users hdid to associate the invite with.</param>
         /// <param name="toEmail">The To email address.</param>
         /// <param name="activationHost">The host of the email validation endpoint.</param>
-        void QueueNewInviteEmail(string hdid, string toEmail, Uri activationHost);
+        /// <param name="existingInviteKey">The key of the existing invite.</param>
+        void QueueNewInviteEmail(string hdid, string toEmail, Uri activationHost, Guid? existingInviteKey = null);
```

Comment:
I dont think the GUID pass as invite key should ever be null

Reply:
updated